<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Admins-MacBook-Pro-3.local" ignored="0" name="Default test" tests="1" failures="1" timestamp="28 Dec 2018 03:58:54 GMT" time="0.882" errors="0">
  <testcase name="LaunchIRCTC" time="0.843" classname="com.coda.Parallel.parallel.SampleClassOne">
    <failure type="org.openqa.selenium.remote.UnreachableBrowserException" message="Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: &amp;apos;3.11.0&amp;apos;, revision: &amp;apos;e59cfb3&amp;apos;, time: &amp;apos;2018-03-11T20:26:55.152Z&amp;apos;
System info: host: &amp;apos;Admins-MacBook-Pro-3.local&amp;apos;, ip: &amp;apos;fe80:0:0:0:ca8:674a:b2b0:3af6%en0&amp;apos;, os.name: &amp;apos;Mac OS X&amp;apos;, os.arch: &amp;apos;x86_64&amp;apos;, os.version: &amp;apos;10.13.4&amp;apos;, java.version: &amp;apos;1.8.0_65&amp;apos;
Driver info: driver.version: RemoteWebDriver">
      <![CDATA[org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.Build info: version: '3.11.0', revision: 'e59cfb3', time: '2018-03-11T20:26:55.152Z'System info: host: 'Admins-MacBook-Pro-3.local', ip: 'fe80:0:0:0:ca8:674a:b2b0:3af6%en0', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.13.4', java.version: '1.8.0_65'Driver info: driver.version: RemoteWebDriverat org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:566)at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:209)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:132)at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:145)at com.coda.Parallel.parallel.SampleClassOne.LaunchIRCTC(SampleClassOne.java:34)at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)at java.lang.reflect.Method.invoke(Method.java:497)at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)at org.testng.internal.Invoker.invokeMethod(Invoker.java:571)at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:707)at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:979)at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)at org.testng.TestRunner.privateRun(TestRunner.java:648)at org.testng.TestRunner.run(TestRunner.java:505)at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)at org.testng.SuiteRunner.run(SuiteRunner.java:364)at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)at org.testng.TestNG.runSuitesSequentially(TestNG.java:1187)at org.testng.TestNG.runSuitesLocally(TestNG.java:1116)at org.testng.TestNG.runSuites(TestNG.java:1028)at org.testng.TestNG.run(TestNG.java:996)at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)Caused by: java.net.ConnectException: Failed to connect to /192.168.100.34:4444at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:240)at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:158)at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:256)at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:134)at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:113)at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:125)at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)at okhttp3.RealCall.execute(RealCall.java:77)at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:101)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:101)at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:73)at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:545)... 29 moreCaused by: java.net.ConnectException: Connection refusedat java.net.PlainSocketImpl.socketConnect(Native Method)at java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:350)at java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:206)at java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:188)at java.net.SocksSocketImpl.connect(SocksSocketImpl.java:392)at java.net.Socket.connect(Socket.java:589)at okhttp3.internal.platform.Platform.connectSocket(Platform.java:125)at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:238)... 51 more]]>
    </failure>
  </testcase> <!-- LaunchIRCTC -->
</testsuite> <!-- Default test -->
